packages/EDA/wasm/ANOVA/ANOVA.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include "../../../../../Eigen/Eigen/Dense"
+#include <emscripten.h>
+
```

Comment:
I think that the following should be added:
1) problem specification, for example, "_... computation of smth the great importance is implemented .._."
2) links to scientific sources with formulas.